src/test/java/net/starlark/java/eval/Benchmarks.java

```diff
@@ -209,19 +229,19 @@ private static void fail(String format, Object... args) {
 
   // Runs benchmark function f for the specified time budget
```

Comment:
Update comment:   "time budget or number of iterations, exactly one of which must be nonnegative".